src/server/routes/rest.js

```diff
@@ -36,6 +45,14 @@ router.get('/get-graph-and-layout', function (req, res) {
   });
 });
 
+// Expose a rest endpoint for controller.getLayoutHistory
+router.get('/get-layout-history', function (req, res) {
```

Comment:
Same thing for this endpoint, it should take in query params that specify how far back in the history you want to go encoded as a number.

Reply:
It now takes in a `numEntries` field.